pynumaflow/shared/server.py

```diff
@@ -207,10 +208,10 @@ async def start_async_server(
     await server_async.start()
 
     # Add the server information to the server info file
-    # Here we just write the protocol and language information
```

Comment:
for start_sync_server, I think we can remove the `if add_info_server` check. Now that we are adding info server file for all funcs.

Reply:
shoudl we also remove the check on line 126 as well? `if server_info and server_info_file` since we provide info file all the time

Reply:
@kohlisid WDYT?

Reply:
Makes sense, I would just take a look at this and share. There can be a couple of refactors that we can take as pointed in the comment on the above thread. Will get back